manual/alshelp.css

```diff
@@ -0,0 +1,68 @@
+#body {
+overflow-y: scroll;
+overflow-x: hidden;
+}
+
+#left_top {
+position: absolute;
+top:0;
+left:0;
+height:8em;
+width:13em;
+background-color:ivory;
+}
+
+#left_bottom {
+position: absolute;;
+top:8em;
+left:0;
+height:100%;
+width:13em;
+background-color:cornsilk;
+}
+
+#right {
+position: absolute;;
+top:0;
+left:13em;
+height:120%;
```

Comment:
I just forgot to clean out most of alshelp.css � some junk, some moved into the alshelp/toc_ABC.html files.

Re: I'd recommend removing the height/width and then setting top, right, bottom to 0 (keeping left at 13em).

This doesn�t appear to work.  When I tried it, things scrunched up & to the left.

� Ken

On Nov 29, 2015, at 4:23 PM, Chuck Houpt <notifications@github.com> wrote:

> In manual/alshelp.css:
> 
> > +}
> > +
> > +#left_bottom {
> > +position: absolute;;
> > +top:8em;
> > +left:0;
> > +height:100%;
> > +width:13em;
> > +background-color:cornsilk;
> > +}
> > +
> > +#right {
> > +position: absolute;;
> > +top:0;
> > +left:13em;
> > +height:120%;
> 
> The 120% height and width setting causes problem as you resize the window. (sometime the "right" frames scroll bar is visible, sometime hidden, etc).
> 
> I'd recommend removing the height/width and then setting top, right, bottom to 0 (keeping left at 13em).
> 
> �
> Reply to this email directly or view it on GitHub.
>